app/views/data_files/multi-steps/_template.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='panel panel-default'>
+  <div class='panel-heading'>
+    <% if Seek::Config.file_templates_enabled %>
+       File template, data type and format
+    <% else %>
+       Data type and format
+    <% end %>
+  </div>
+
+  <div class='panel-body'>
+    <% if Seek::Config.file_templates_enabled %>
+      <%= render :partial => "assets/select_template", :locals => {:resource => (@data_file ? @data_file : DataFile.new),
+      :templates => FileTemplate.all } %>
```

Comment:
Should `FileTemplate.all` be scoped somehow, or have authorization performed?

Reply:
Probably. Raise as issue?

Reply:
Now scoped. The presentation of the select could be much nicer.